src/arrayMethodSort.js

```diff
@@ -4,8 +4,32 @@
  * Implement method Sort
  */
 function applyCustomSort() {
-  [].__proto__.sort2 = function(compareFunction) {
-    // write code here
+  [].__proto__.sort2 = function (compareFunction) {
+    if (compareFunction === undefined) {
+      for (let i = 0; i < this.length - 1; i++) {
+        if (this[i] + '' > this[i + 1] + '') {
```

Comment:
Using '+' for string conversion is not explicit. It's better to use String() for clarity and to avoid coercion issues.

Reply:
Agree with this comment - please, use `String(this[i])` instead of concatenating to transform value to string
It will be way easier to read